ovs/lib/storagerouter.py

```diff
@@ -755,6 +798,27 @@ def add_vpool(parameters):
                     root_client.run('virsh pool-start {0}'.format(vpool_name))
                     root_client.run('virsh pool-autostart {0}'.format(vpool_name))
 
+        # Start ovs ceph proxy BEFORE starting volumedriver
+        if vpool.backend_type.code == 'ceph_ovs_proxy':
+            params = {'PORT': str(ceph_proxy_port),
+                      'CEPHCONF': ceph_proxy_config_file,
+                      'VPOOL_NAME': vpool.name}
+            ceph_proxy_service = "ovs-cephproxy_{0}".format(vpool.name)
+            ServiceManager.add_service("ovs-cephproxy", params=params, client=root_client, target_name=ceph_proxy_service)
+            ServiceManager.enable_service(ceph_proxy_service, client=root_client)
+            ServiceManager.start_service(ceph_proxy_service, client=root_client)
+            logger.debug('Starting ceph proxy service {0}'.format(ceph_proxy_service))
```

Comment:
We don't need to start the ceph service. If configured correctly, it should be a startup dependency of the volumedriver. See alba proxy logic